conda/environment-test-3.10.yml

```diff
@@ -1,14 +1,23 @@
-name: cunumeric-test
+name: legate-core-test
 channels:
   - conda-forge
-  - defaults
 dependencies:
   - python=3.10
 
   # build
-  - cutensor>=1.3.3
+  - git
   - nccl
+  - make
+  - zlib
+  - cmake>=3.24
+  - ninja
+  - openmpi
+  - c-compiler
+  - cxx-compiler
+  - gcc_linux-64 # [linux64]
```

Comment:
Do we need to specify gcc specifically? Especially if later we request llvm-openmp

Reply:
I don't know, see also: https://github.com/nv-legate/legate.core/pull/323#discussion_r957830930

Reply:
IIRC, the compiler does not necessarily come with a specific openmp implementation, and package maintainers/users have a choice of which openmp implementation to use.